### PR TITLE
new endpoint to trigger batchjob

### PR DESCRIPTION
**Pull request by cheezi0747**

### CODE REVIEW

File: src/main/java/se/sundsvall/snailmail/api/SnailMailResource.java [hunk 1]
```diff
@@ -19,12 +23,13 @@
 import io.swagger.v3.oas.annotations.media.Schema;
 import io.swagger.v3.oas.annotations.responses.ApiResponse;
 import io.swagger.v3.oas.annotations.tags.Tag;
-import jakarta.validation.Valid;
 
 @RestController
 @Validated
 @RequestMapping("/send/snailmail")
 @Tag(name = "SnailMailSender", description = "SnailMailSender")
+@ApiResponse(responseCode = "400", description = "Bad Request", content = @Content(schema = @Schema(implementation = Problem.class)))
```
Comment (josc04) [inline src/main/java/se/sundsvall/snailmail/api/SnailMailResource.java:31]: Change to `schema = @Schema(oneOf = { Problem.class, ConstraintViolation.class })`?

File: src/test/java/se/sundsvall/snailmail/api/SnailMailResourceTest.java [hunk 2]
```diff
@@ -1,39 +1,100 @@
...
+			.uri("/send/snailmail")
+			.bodyValue(SendSnailMailRequest.builder()
+				.withDepartment("department")
+				.build())
+			.exchange()
+			.expectStatus()
+			.isOk();
 
+		// VERIFY
 		verify(mockSnailMailService, times(1)).sendSnailMail(any(SendSnailMailRequest.class));
```
Comment (josc04) [inline src/test/java/se/sundsvall/snailmail/api/SnailMailResourceTest.java:53]: `times(1)` is redundant and can be removed (goes for all places where it is used)

File: src/test/java/se/sundsvall/snailmail/api/SnailMailResourceTest.java [hunk 3]
```diff
@@ -1,39 +1,100 @@
 package se.sundsvall.snailmail.api;
 
 import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.groups.Tuple.tuple;
 import static org.mockito.ArgumentMatchers.any;
-import static org.mockito.Mockito.doNothing;
 import static org.mockito.Mockito.times;
 import static org.mockito.Mockito.verify;
-import static se.sundsvall.snailmail.TestDataFactory.buildSendSnailMailRequest;
```
Comment (josc04) [inline src/test/java/se/sundsvall/snailmail/api/SnailMailResourceTest.java:8]: Remove `buildSendSnailMailRequest`-method in `TestDataFactory` (if it is not used elsewhere than here)?

Comment (cheezi0747) [reply src/test/java/se/sundsvall/snailmail/api/SnailMailResourceTest.java:8]: Its used elsewhere.

**Review by josc04 — APPROVED** [review]

**Pull request merged**